test/new-e2e/tests/agent-subcommands/status/status_common_test.go

```diff
@@ -99,12 +99,6 @@ func fetchAndCheckStatus(v *baseStatusSuite, expectedSections []expectedSection)
 
 func (v *baseStatusSuite) TestDefaultInstallStatus() {
 	expectedSections := []expectedSection{
-		{
-			name:             `Agent \(.*\)`, // TODO: verify that the right version is output
-			shouldBePresent:  true,
-			shouldContain:    []string{"FIPS Mode: not available"},
-			shouldNotContain: []string{"FIPS proxy"},
-		},
```

Comment:
It would be simpler to use a function return rather than create a custom entry and append to the lists.

```
func (v *baseStatusSuite) shouldContainsFips() {
	if v.Env().Agent.FIPSEnabled {
		return []string{"FIPS Mode: enabled"}
	return FIPSSection.shouldContain = []string{"FIPS Mode: not available"}
}

// [...]
shouldContain:    shouldContainFips()
// [...]
```

WDYT ?